### PR TITLE
Feature- My Profile UI 

### LINKED ISSUE #66: My Profile UI
**Complete By: 30th April 2018, EOD.**

This is the profile page where users can edit their profile details anytime.

Changes required in : **activity_user_profile.xml**

Requirements:

- Name (Editable)
- Slack Handle (Editable)
- Photo/Avatar (Changeable)
    A) Users can either select an avatar from a list of 10-20 avatars
    B) OR Use the image fetched after the Gmail login.
    Note: Feature for manually uploading images (from device) is not required.
- Track (Not Editable) (Should have only “Android Developer Track” for now) 
- Email (Not editable)
- Submit/Next Button

Sample UI link: https://github.com/UdacityAndroidDevScholarship/quiz-app/wiki/App-Mockups#see-user-profile-activity

### PR DESCRIPTION
**Pull request by YuganshT79**
Fixes #66 

Profile UI screen according to the given conditions.
- Editable name, slack handle
- **Edit icon** to select/edit user's profile image
- Uneditable Email, scholarship track

## Before & After Screenshots

**BEFORE**:
Before

**AFTER**:
After

## Other changes (e.g. bug fixes, UI tweaks, refactors)
- Updated `androidTestImplementation`
- Added `CircularImageView` dependency
- Updated `classpath` to latest version

### CODE REVIEW

**Comment by sagarsrao** [comment]
@YuganshT79 Great work in UI.Please fix your conflicting issues.Do not modify project level classpath and retain the same(build.gradle(project))

File: app/build.gradle [hunk 1]
```diff
@@ -55,8 +55,8 @@ dependencies {
     implementation "com.google.firebase:firebase-messaging:$rootProject.firebaseVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
     testImplementation "junit:junit:$rootProject.junitVersion"
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    androidTestImplementation 'com.android.support.test:runner:1.0.1'
```
Comment (vansh1sh) [inline app/build.gradle]: Why not 1.0.2?

File: build.gradle [hunk 2]
```diff
@@ -7,12 +7,12 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```
Comment (vansh1sh) [inline build.gradle]: @YuganshT79 Please don't make changes in the project level build.gradle file.

File: app/build.gradle (hunk 1 above)
Comment (YuganshT79) [reply app/build.gradle]: I previously set it to 1.02 but @sagarsrao said I should not modify the build.gradle file so I changed them back to 1.0.1

File: build.gradle (hunk 2 above)
Comment (YuganshT79) [reply build.gradle]: What was the conflict in the Pull request then ?

File: app/build.gradle (hunk 1 above)
Comment (vansh1sh) [reply app/build.gradle]: Okay

Comment (YuganshT79) [reply app/build.gradle]: What should I do?
Should I change it back to 1.0.2?
Also should I push the updated UI to the branch with **Google Udacity Scholars** text and India Logo?

File: build.gradle (hunk 2 above)
Comment (vansh1sh) [reply build.gradle]: Probably there must be multiple lines, out of which you removed the wrong one.

File: app/build.gradle (hunk 1 above)
Comment (vansh1sh) [reply app/build.gradle]: @YuganshT79 Yes change it to 1.0.2. Just do a git pull, or compare your recent code with the latest commit in the branch. This way there won't be any conflicts.

Comment (YuganshT79) [reply app/build.gradle]: Ok, will review the changes and Push the code, and what about the updated UI code ?

Comment (vansh1sh) [reply app/build.gradle]: What exactly do you mean by updated UI code? If there's some updates in the profile page code (xml file) from your end, include that as well in the commit.

Comment (YuganshT79) [reply app/build.gradle]: Sure, will do 👍

File: app/build.gradle [hunk 3]
```diff
@@ -56,14 +56,15 @@ dependencies {
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
     testImplementation "junit:junit:$rootProject.junitVersion"
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
```
Comment (vansh1sh) [inline app/build.gradle]: @YuganshT79 This will remain 3.0.1

File: build.gradle (hunk 2 above)
Comment (vansh1sh) [reply build.gradle]: This will be 3.1.2

**Comment by vansh1sh** [comment]
Looks good now.

**Pull request merged**